app/views/pageflow/social_share/_image_tags.html.erb

```diff
@@ -1,5 +1,7 @@
 <% image_urls.each do |image_url| %>
   <%= tag :meta, property: "og:image", content: social_share_normalize_protocol(image_url) %>
+  <%= tag :meta, property: "og:image:width", content: 1200 %>
+  <%= tag :meta, property: "og:image:height", content: 630 %>
```

Comment:
Where do these values come from? The `medium` style is defined as `1024x1024>`. I think we will need a dynamic method to calculate the correct size. The dimensions of the original image are also stored on `ImageFile`. Maybe Facebook only cares about the ratio - not the actual size?

Reply:
I am not sure how facebook uses these width height parameters. they have mentioned this 
> 
Use og:image:width and og:image:height Open Graph tags
Using these tags will specify the image dimensions to the crawler so that it can render the image immediately without having to asynchronously download and process it.
> 
from my observation I could not see any difference in post rendering when specifying different width and height
I found the above values 1200x600 by just searching the most optimum values for facebook image share.
We can  already have mechanism to get image dimensions in image_file.rb save_image_dimensions we can apply that?

Reply:
applied actual image width and height from ImageFile